docs/docs/guides/migrate/sources/auth0-guide.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Migrating Users from Auth0 to ZITADEL (Including Password Hashes)
+sidebar_label: Auth0 Migration Guide
+---
+
+## 1. Introduction
+
+**Why migrate from Auth0 to ZITADEL?**  
+This guide will walk you through the steps to migrate users from Auth0 to ZITADEL, including password hashes (with Auth0 support assistance), so users don't need to reset their passwords.
+
+**What you'll learn with this guide**
+- How to prepare your data from Auth0
+- Use of ZITADEL migration tooling
+- Performing the user import via API
+- Troubleshooting and validating the migration
+
+---
+
+## 2. Prerequisites
+
+### 2.1. Install Go
+The migration tool is written in Go. Download and install the latest version of Go from the [official Go website](https://go.dev/).
+
+### 2.2. Create a ZITADEL Instance and Organization
+You'll need a target organization in ZITADEL to import your users. You can create a new organization or use an existing one.  
+
+If you don't have one, you can [sign up for free here](https://zitadel.com).  
+See: [Managing Organizations in ZITADEL](https://docs.zitadel.com/).
+
+> **Note:** Copy your Organization ID (Resource ID) for use in later steps.
+
+---
+
+## 3. Preparing Auth0 Data
+
+### 3.1. Export User Profiles and Password Hashes from Auth0
+You cannot bulk export user data from the Auth0 Dashboard. Instead, use the Auth0 Management API or the User Import/Export extension.
+
+> **Important:** Password hashes cannot be obtained in a self-service way.  
+> You must open a **support ticket** with Auth0 and request a password hash export.  
+> If approved, Auth0 will provide an export containing the password hashes.  
+
+Reference: [Export hashed passwords from Auth0](https://auth0.com/docs/users/export-users/export-users-passwords)
+
+---
+
+## 4. Running the ZITADEL Migration Tool
+
+### 4.1. Install the Migration Tool
+Follow the installation instructions to set up the ZITADEL migration tool from [ZITADEL Tools](https://github.com/zitadel/zitadel-tools).
+
+### 4.2. Generate Import JSON
+Use the migration tool to convert the Auth0 export file into a ZITADEL-compatible JSON.
+
+**Example workflow:**
+
+Sample `passwords.ndjson`:
+```json
+{"_id":{"$oid":"emxdpVxozXeFb1HeEn5ThAK8"},"email_verified":true,"email":"tommie_krajcik85@hotmail.com","passwordHash":"$2b$10$d.GvZhGwTllA7OdAmsA75uGGzqr/mhdQoU88M3zD.fX3Vb8Rcf33.","password_set_date":{"$date":"2025-06-30T00:00:00.000Z"},"tenant":"test","connection":"Username-Password-Authentication","_tmp_is_unique":true}
+```
```

Comment:
Is this the sample from auth0 or zitadel? We should make this clear.

Reply:
Changed it to:
`Sample `passwords.ndjson` content, as obtained from the Auth0 Support team:`